make_list_flat.py

```diff
@@ -0,0 +1,13 @@
+dim_list = [[[2]], [4, [5, 6, [6], 6, 6, 6], 7]]
+
+def flattern(array : list):
```

Comment:
array - not good name for var, since is also name of datatype
use something like 'data', 'items' etc.